modules/update-service-install-web-console.adoc

```diff
@@ -18,20 +18,22 @@ Enter `Update Service` into the *Filter by keyword...* field to find the Operato
 
 . Choose *OpenShift Update Service* from the list of available Operators, and click *Install*.
 
-.. Channel `v1` is selected as the *Update Channel* since it is the only channel available in this release.
+.. Select an *Update channel*.
+
+.. Select a *Version*.
```

Comment:
Not related to this web console procedure, but is there also an option now to specify a version when [installing this Operator via the CLI](https://docs.openshift.com/container-platform/4.14/updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.html#update-service-install-cli_updating-restricted-network-cluster-osus)? If so, I can also update that procedure either in this PR or in another one.

Reply:
This is a doc about installing a specific version operator. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/administrator-tasks#olm-installing-specific-version-cli_olm-adding-operators-to-a-cluster